cli/ts/utils/verifiers.ts

```diff
@@ -0,0 +1,91 @@
+import { Contract } from "ethers";
+import { genTreeProof } from "maci-crypto";
+
+/**
+ * Loop through each per vote option spent voice credits and verify it on-chain
+ *
+ * @param tallyContract The tally contract
+ * @param tallyData The tally.json file data
+ * @param voteOptionTreeDepth The vote option tree depth
+ * @param newSpentVoiceCreditsCommitment The total spent voice credits commitment
+ * @param newResultsCommitment The tally result commitment
+ *
+ * @returns list of the indexes of the tally result that failed on-chain verification
+ */
+export const verifyPerVOSpentVoiceCredits = async (
+  tallyContract: Contract,
+  tallyData: any,
```

Comment:
please use the tally data interface here https://github.com/privacy-scaling-explorations/maci/blob/dev/cli/ts/utils/interfaces.ts#L25